Barotrauma/BarotraumaShared/SharedSource/StatusEffects/StatusEffect.cs

```diff
@@ -616,6 +616,9 @@ public IEnumerable<Explosion> Explosions
 
         private readonly List<ItemSpawnInfo> spawnItems;
 
+        private readonly String forceSayIdentifier = null;
+        private readonly bool forceSayInRadio;
```

Comment:
These should be commented to allow them to be properly listed in the autogenerated modding documentation.

Reply:
Is there any way I can also add an automated description to the newly added "ForceSay" command as a whole or does that need to be done manually?

Reply:
We have a script that manually parses the comments from the code for the modding doc, so these comments you added will do just fine. :)